src/app/products/taxonomy.ts

```diff
@@ -1,7 +1,8 @@
-
+import { Product } from './product';
 export class Taxonomy {
-    taxonomyId: number;
-    name: string;
-    type: string;
-    subType: string;
+  taxonomyId: number;
+  name: string;
+  type: string;
+  subType: string;
+  product: Product;
```

Comment:
remove this line, taxonomy doesnt have a product

Reply:
donw